block/aggregation_test.go

```diff
@@ -201,12 +201,7 @@ func TestAggregationLoop_Normal_PublishBlockError(t *testing.T) {
 	defer publishLock.Unlock()
 
 	calls := publishCalls.Load()
-	assert.GreaterOrEqualf(calls, int64(4), "publishBlock should have been called multiple times (around 4), but was called %d times", calls)
-	assert.LessOrEqualf(calls, int64(5), "publishBlock should have been called multiple times (around 4-5), but was called %d times", calls)
-
-	require.GreaterOrEqual(len(publishTimes), 3, "Need at least 3 timestamps to check intervals after error")
-	for i := 2; i < len(publishTimes); i++ {
-		interval := publishTimes[i].Sub(publishTimes[i-1])
-		WithinDuration(t, blockTime, interval, tolerance)
-	}
+	require.Equal(calls, int64(1))
+	require.ErrorContains(<-errCh, expectedErr.Error())
+	require.Equal(len(publishTimes), 1, "Expected only one publish time after error")
```

Comment:
The relevant publishing issues that shouldn't halt have been changed to warning logs.